Models/models.js

```diff
@@ -12,7 +12,19 @@ async function getApiModel() {
     return JSON.parse(documentation);
 }
 
+async function getAllArticlesModel() {
+    const results = await db.query(`
+    SELECT articles.article_id, articles.author, articles.topic, articles.created_at,
+articles.votes, articles.title, articles.article_img_url,
+COUNT(comments.article_id) as comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;
+    `);
+    if (!results.rows.length) {
+        throw({code: 404});
```

Comment:
Instead of throwing an error please use Promise.reject so that your error handlers can catch it

Reply:
You've also not written any tests for this error - which needs to happen